### PR TITLE
fix(popover): add popoverTrigger Input

### PR DESCRIPTION
**Pull request by mathsusan**
## Description

adding popoverTrigger which was not there

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the **CONTRIBUTING** document.
- [ x] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/popover/tests/popover.component.spec.ts [hunk 1]
```diff
@@ -37,15 +37,15 @@ describe('Popover Test', () => {
...
+    testEl.triggerEventHandler('click', null);
     fixture.detectChanges();
   }));
 
   it('should match snapshot', () => {
     expect(overlayContainerEl).toMatchSnapshot();
   });
 
-  it('popover should appear when hovering over badge NOTE: it uses tooltip', () => {
+  it('popover should appear when cklick over badge', () => {
```
Comment (bfbiggs) [inline angular/src/lib/popover/tests/popover.component.spec.ts]: spelling

File: angular/src/docs/app/playground/playground.component.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Component, OnInit } from '@angular/core';
     <div className="row">
       <div className="example-spacing">
         <!-- Insert Example Here -->
+        <example-popover-trigger></example-popover-trigger>
```
Comment (bfbiggs) [inline angular/src/docs/app/playground/playground.component.ts]: clean up playground

**Comment by bfbiggs** [comment]
Can you rename your commit message to 'fix(popover): add popoverTrigger Input'

**Comment by mathsusan** [comment]
> I tested it in the playground, we may need an additional prop or something that will keep the popover open when clicking inside of the popover. For instance, we may have a list/selects inside the popover that would need to stay visible after clicking. Clicking outside of the popover will close the popover.
> 
> https://drive.google.com/file/d/1bXlQytGUzwf-UsZwIHwHv_8PwhJU2JSf/view
> 
> Pretty much how the click example works in React.
> 
> https://stage.momentum.design/components/popover/code
> @mathsusan @pauljeter @bfbiggs

This definitely sounds like a feature request it was not part of the original requirements and should be put on the backlog.

**Comment by bfbiggs** [comment]
I think the intent of the task was to allow it to have the same functionality as React. The props Daani is referencing are probably closeOnClick and allowClickAway that actually live in the EventOverlay component in React but since we're using the CDK in Angular, probably need to live in the Popover. Let's merge and we can revisit any additional needs in planning.

**Pull request merged**